pkgs/development/compilers/dotnet/default.nix

```diff
@@ -22,6 +22,7 @@ rec {
 
   # v3.1 (lts)
   aspnetcore_3_1 = buildAspNetCore {
```

Comment:
I believe these entries get automatically generated with the update script so that you can just copy paste them in here, we might want to add the `inherit icu` line there as well.

Reply:
Fixed print-hashes.sh